cloudify/context.py

```diff
@@ -712,6 +712,10 @@ def workflow_id(self):
         """
         return self._context.get('workflow_id')
 
+    @property
+    def host_ip(self):
```

Comment:
are you sure `host_ip` is a good name? we already have a `ctx.instance.host_ip` that means the ip of the instance, do you want to use the same name for another thing?

Reply:
Well, the reason of choosing such name is because `_get_current_context()` would be either `CloudifyContext` instance or `CloudifyWorkflowContex` or any instances extend `_WorkflowContextBase`

Reply:
or we can change it to something else maybe so that it would be common between all type of contex

Reply:
@mabuaisha yeah after more thought i think i would change it
"host ip", we deal with 100000 hosts, so it's not easy to guess that the "host" means "manager". So probably just "manager_ip" is better
(also it doesn't have to be an ip of course, it can be a hostname, so maybe manager_addr or something, not sure)

Reply:
yeah `rest_host` is a better name